### PR TITLE
Add Hebrew Translation

### PR DESCRIPTION
**Pull request by yotam180**
#1

### CODE REVIEW

File: pkg/colorful-hebrew.man [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.\" דף הוראות של Colorful
+.\" צרו קשר עם veg@svgames.pl לתיקוני בעיות או שגיאות.
+.TH man 6 "2014-03-16" "1.2" "Games Manual"
```
Comment (suve) [inline pkg/colorful-hebrew.man]: "Games Manual" should be translated. The date should also reflect the manpage modification date.

File: pkg/colorful-hebrew.man [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+colorful - משחק יריות פשוט בדו-מימד.
+.SH SYNOPSIS
+\fBcolorful\fR
+.SH DESCRIPTION
+Colorful הוא משחק פשוט יריות פשוט דו-מימדי, בו הפרוטגוניסט מטייל במבוך של 
+מערות ומסדרונות במטרה לאסוף יצירות צבע.
+.SH CONTROLS
+המשחק משוחק בעזרת המקלדת בלבד. הגדרות החצים במתכונת ברירת מחדל:
+.TP
+\fBArrow keys\fR
```
Comment (suve) [inline pkg/colorful-hebrew.man]: "Arrow keys" should also be translated.

File: pkg/colorful-hebrew.man [hunk 3]
```diff
@@ -0,0 +1,31 @@
+.\" דף הוראות של Colorful
+.\" צרו קשר עם veg@svgames.pl לתיקוני בעיות או שגיאות.
+.TH man 6 "2014-03-16" "1.2" "Games Manual"
+.SH NAME
```
Comment (suve) [inline pkg/colorful-hebrew.man]: All the headers (`.SH`) should also be translated.

**Review by suve — CHANGES_REQUESTED** [review]
Hello and thank you for your interest. I noticed a couple of untranslated strings. Please address those and it'll be good to merge.

File: pkg/colorful-hebrew.man (hunk 2 above)
Comment (yotam180) [reply pkg/colorful-hebrew.man]: Alright. Gimme a couple of minutes for that.

**Comment by yotam180** [comment]
@suve Updated the untranslated strings. I think it was updated in the pull request also (right...?)
Any other things to correct?

**Review by suve — APPROVED** [review]

**Comment by suve** [comment]
Nope, everything's good now. Merged. Thanks!

**Pull request merged**